File: belleza.py
import Productos.Productos

class Belleza(Productos):
    def __init__(self, nombre, precio, fecha_de_importacion, stock, color):
        super().__init__(nombre, precio, fecha_de_importacion, stock)
        self.color = color
        self.productos_belleza = []

    def registrar_producto_belleza(self, nombre, precio, fecha_de_importacion, stock):
        print("Ingrese el color del producto de belleza:")
        self.color = input()
        nuevo_producto_belleza = Belleza(nombre, precio, fecha_de_importacion, stock, self.color)
        self.productos_belleza.append(nuevo_producto_belleza)

File: src/Productos/belleza.java
package Productos;
import java.util.Scanner;
import java.util.ArrayList;
public class belleza extends Productos{
    ArrayList <belleza> productosBelleza=new ArrayList<belleza>();
    Scanner leer=new Scanner(System.in);
    private String color;
    public belleza(String nombre, double precio, String fechaDeImportacion, int stock, String color) {
        super(nombre, precio, fechaDeImportacion, stock);
        this.color=color;

    }
    public void RegistrarProductoBelleza(String nombre,double precio, String fechaDeImportacion, int stock){
        System.out.println("ingrese la fecha de caducidad con la siguiente estructura: dia/mes/año, ejepmlo: 06/10/24");
        color=leer.nextLine();
        belleza belleza=new belleza(nombre, precio, fechaDeImportacion, stock,color);
        productosBelleza.add(belleza);
    }
}
